rest_framework_filters/filterset.py

```diff
@@ -70,6 +70,17 @@ def related_filters(self):
             ])
         return self._related_filters
 
+    @property
+    def expanded_filters(self):
+        if '_expanded_filters' not in self.__dict__:
+            self._expanded_filters = self.base_filters.copy()
+            for filter_name, f in self.related_filters.items():
+                del self._expanded_filters[filter_name]
```

Comment:
The related filters shouldn't be deleted (they're functional `ModelChoiceFilter`s).

Reply:
I missed this one. Thank you!